src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,134 @@
+package homework_2.random_chars_table;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.Locale;
+
+public class RandomCharsTable {
+    private final String ERR_MSG = "Passed parameters should match the format [positive integer] [positive integer] [even|odd]";
+
+    public void run() {
+
+        RandomCharsTable table = new RandomCharsTable();
+        String lenStr;
+        String widStr;
+        String method;
```

Comment:
Opt: this way of initialization is rarely used in Java

Reply:
change to single line init